Cargo.toml

```diff
@@ -112,11 +112,12 @@ native-tls-crate = { version = "0.2.8", optional = true, package = "native-tls"
 tokio-native-tls = { version = "0.3.0", optional = true }
 
 # rustls-tls
-hyper-rustls = { version = "0.22.1", default-features = false, optional = true }
-rustls = { version = "0.19", features = ["dangerous_configuration"], optional = true }
-tokio-rustls = { version = "0.22", optional = true }
+hyper-rustls = { version = "0.23", default-features = false, optional = true }
+rustls = { version = "0.20", features = ["dangerous_configuration"], optional = true }
+tokio-rustls = { version = "0.23", optional = true }
 webpki-roots = { version = "0.21", optional = true }
-rustls-native-certs = { version = "0.5", optional = true }
+rustls-native-certs = { version = "0.6.0", optional = true }
+rustls-pemfile = { version = "0.2.1", optional = true }
```

Comment:
Do we need to specify the patch version?
Maybe just
`rustls-pemfile = { version = "0.2", ..`
`rustls-native-certs = { version = "0.6", ...`

Reply:
That's just a stylistic difference though, it doesn't make any difference. I will usually omit any `.0` suffixes but use the latest version released at the time I write up the dependency.